smdebug/core/utils.py

```diff
@@ -317,7 +317,8 @@ def get_tb_worker():
 
 
 def get_distributed_worker():
-    """Get the rank for horovod or torch distributed. If none of them are being used,
+    """This fn should only be used for torch.
```

Comment:
Can we enforce this with something stronger than a comment?
Check if we're running torch for example?

Reply:
Actually my comment was wrong, but updated this function to work with SMP as well. And made all imports happen only once